src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.wasm.cs

```diff
@@ -156,6 +157,20 @@ protected override Size ArrangeOverride(Size finalSize)
 
 		partial void OnFontWeightChangedPartial() => _fontWeightChanged = true;
 
+		private void UpdateText()
+		{
+			if (_run is null)
+			{
+				_run = new Run();
+				_run.SetText(Text);
+				AddChild(_run);
```

Comment:
Shouldn't we reset/remove the run when Text is set first, then Inlines is used ? Let's add a test for that, it's likely to break.

Reply:
@jeromelaban It looks like when I access `Inlines`, this will clear all children.

![image](https://user-images.githubusercontent.com/31348972/190889647-f4cac3ec-d628-43ca-a6b0-77d8bc3b4b9a.png)

Reply:
Got it! Can you still add a test that validates that behavior? Seems a bit too implicit :) You can validate explicitly for Wasm, but poking a children explicitly.